recipes/ccache-toolchain/activate.sh

```diff
@@ -2,3 +2,11 @@
 
 export _OLD_PATH=${PATH}
 export PATH="${PREFIX}/bin/conda_forge_ccache:${PATH}"
+
+if [ -f "/feedstock_root/build_artefacts" ]; then
+    export CCACHE_DIR=/feedstock_root/build_artefacts/.ccache
+fi
+
+# Set max cache size so we don't carry old objects for too long
+ccache -M 400M
```

Comment:
@gqmelo, for sagelib I need around 350M on osx. That's why I had it like this. What would happen if this is set to 200M and in sagelib package it is set to 400M? Will setting to 200M here, immediately downsize to 200M?